<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFastboot</class>
 <widget class="QDialog" name="DialogFastboot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFastbootFile">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Fastboot location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFastbootFileStatus">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/icons/icons/red_cross.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineFastbootFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushBrowseFastboot">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelImageFile">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Image file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImageFileStatus">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/icons/icons/red_cross.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineImageFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushBrowseImage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="30,70">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelPartition">
         <property name="text">
          <string>Partition to flash</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="boxPartition"/>
       </item>
       <item>
        <widget class="QLabel" name="labelDevices">
         <property name="text">
          <string>Devices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listDevices"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelOutput">
         <property name="text">
          <string>Fastboot output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textOutput"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="labelCommandLine">
       <property name="text">
        <string>Command line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCommandLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushFlash">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Flash</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
